tests/overlay-non-existent-destination/overlay-non-existent-destination.yaml

```diff
@@ -1,6 +1,12 @@
 architecture: amd64
 
 actions:
+  # This overlay action is expected to error out here because the source
+  # doesn't exist in the recipe directory.
```

Comment:
so now the next part is no longer tested because this one already errored out right or am i missing something?